server/glusterfsd.py

```diff
@@ -195,8 +198,8 @@ def start_args():
     set_volume_id_xattr(brick_path, volume_id)
 
     volfile_id = "%s.%s.%s" % (volname, nodename, brick_path_name)
-    volfile_path = os.path.join(VOLFILES_DIR, "%s.vol" % volfile_id)
-    generate_brick_volfile(volfile_path, volname, volume_id, brick_path)
+    storage_unit_volfile_path = os.path.join(VOLFILES_DIR, "%s.vol" % volfile_id)
+    generate_brick_volfile(storage_unit_volfile_path, volname, volume_id, brick_path)
```

Comment:
Also generate Client volfile here. That will be used to serve when volfile-server option is used.

Save generated client volfiles to /var/lib/kadalu/volfiles/<volname>.vol because that is the directory used by volfile server. Ref: https://github.com/kadalu/volgen/blob/main/templates/storage_unit.vol.j2#L60

Reply:
Updated shd, brick & client volfiles to above dir for uniformity.